pkg/libs/endpointaccessible/endpoint_accessible_controller.go

```diff
@@ -46,13 +55,15 @@ func NewEndpointAccessibleController(
 		endpointListFn:         endpointListFn,
 		getTLSConfigFn:         getTLSConfigFn,
 		availableConditionName: name + "EndpointAccessibleControllerAvailable",
+		maxCheckLatency:        resyncInterval - 5*time.Second,
```

Comment:
The only reason we need this offset is to avoid coincidentally skipping a sync -- there's no point to make it relative, as we just need to be several seconds off, no matter what the resync interval is.

Reply:
Sure sure. Not a real concern.